conf/log4j.properties

```diff
@@ -12,63 +12,73 @@
+log4j.logger.com.zimbra.qa.selenium.framework.util.HarnessException=A3
 
 # A2 is set to be a FileAppender.
-log4j.appender.A3=org.apache.log4j.FileAppender
-log4j.appender.A3.File=${outputDirectory}/testcase.txt
-log4j.appender.A3.MaxFileSize=100KB
-log4j.appender.A3.Append=false
+appender.A3.type=org.apache.log4j.RollingFileAppender
+appender.A3.fileName=${outputDirectory}/selenium.txt
+appender.A3.policies.type=Policies
+appender.A3.policies.size.type=SizeBasedTriggeringPolicy
+appender.A3.policies.size.size=100KB
+appender.A3.append=false
 
 # A2 uses PatternLayout.
-log4j.appender.A3.layout=org.apache.log4j.PatternLayout
-log4j.appender.A3.layout.ConversionPattern=%m%n
+appender.A3.layout.type=PatternLayout
+appender.A3.layout.pattern=m%n
 
-log4j.appender.general=org.apache.log4j.FileAppender
-log4j.appender.general.File=${outputDirectory}/debug.txt
-log4j.appender.general.Threshold=WARN
-log4j.appender.general.append=false
-log4j.appender.general.layout=org.apache.log4j.PatternLayout
-log4j.appender.general.layout.ConversionPattern=%-4r %-5p %c %x - %m%n
+appender.general.type=org.apache.log4j.RollingFileAppender
+appender.general.fileName=${outputDirectory}/debug.txt
+appender.general.level=WARN
+appender.general.append=false
+appender.general.layout=org.apache.log4j.PatternLayout
+appender.general.layout.ConversionPattern=%-4r %-5p %c %x - %m%n
 
 log4j.logger.com.zimbra.qa.selenium.framework.util.staf.StafAbstract=INFO,coverage
 log4j.logger.com.zimbra.qa.selenium.framework.util.CodeCoverage=INFO,coverage
 log4j.logger.com.zimbra.qa.selenium.framework.core.ExecuteHarnessMain.MethodListener=INFO,coverage
-log4j.appender.coverage=org.apache.log4j.FileAppender
-log4j.appender.coverage.File=${outputDirectory}/coverage.txt
-log4j.appender.coverage.append=false
-log4j.appender.coverage.layout=org.apache.log4j.PatternLayout
-log4j.appender.coverage.layout.ConversionPattern=%-4r %-5p %c %x - %m%n
+
+appender.coverage.type=org.apache.log4j.RollingFileAppender
+appender.coverage.fileName=${outputDirectory}/coverage.txt
+appender.coverage.level=WARN
+appender.coverage.append=false
+appender.coverage.layout=org.apache.log4j.PatternLayout
+appender.coverage.layout.ConversionPattern=%-4r %-5p %c %x - %m%n
 
 log4j.logger.com.zimbra.qa.selenium.framework.util.SleepMetrics=INFO,sleep
-log4j.appender.sleep=org.apache.log4j.FileAppender
-log4j.appender.sleep.File=${outputDirectory}/sleep.txt
-log4j.appender.sleep.append=false
-log4j.appender.sleep.layout=org.apache.log4j.PatternLayout
-log4j.appender.sleep.layout.ConversionPattern=%-4r - %m%n
+appender.sleep.type=org.apache.log4j.RollingFileAppender
+appender.sleep.fileName=${outputDirectory}/sleep.txt
+appender.sleep.level=INFO
+appender.sleep.append=false
+appender.sleep.layout=org.apache.log4j.PatternLayout
+appender.sleep.layout.ConversionPattern=%-4r - %m%n
 
 # Set selenium to WARN
-log4j.logger.org.openqa=WARN
\ No newline at end of file
+rootLogger.level = WARN
```

Comment:
also define a default appender to rootLogger. 
rootLogger.appenderRef.stdout.ref = STDOUT